pact/coin-contract/v4/coin-v4.repl

```diff
@@ -0,0 +1,162 @@
+(env-data
+  { "coin-file": "../v4/coin-v4.pact"
+  , "coin-load-gas": 3413
+  })
+(load "../v2/coin.repl")
+(load "../v3/coin.repl")
```

Comment:
Instead of calling other repls, let's copypasta them all into here and get rid of the switching/gas parameterization. In other words, v3 is already below, so just delete the above lines and paste v2 repl in here, and get it passing, and that will be our baseline.

Reply:
Done